src/main/java/codeu/model/data/Profile.java

```diff
@@ -4,7 +4,7 @@
 
 /** Class representing a user's profile */
 public class Profile {
-	private final UUID author;
+	private final UUID user;
```

Comment:
What was the reason behind switching names (just curious)? Will it be to differentiate between the person who wrote (and can edit) the profile and other users just viewing the profile?

Reply:
I think it would be more clear to have it be "user" instead of "author" since it sounded better in general. That is just me though.

Reply:
That's fair. Thanks!